config/runtime/v4l2-decoder-conformance.jinja2

```diff
@@ -0,0 +1,4 @@
+{%- set boot_commands = 'nfs' %}
+{%- set test_method = 'v4l2-decoder-conformance' %}
+{%- set base_template = 'base/' + runtime + '.jinja2' %}
+{%- extends base_template %}
```

Comment:
This is very similar to some of the other templates like `kselftest`... At some point we should probably make `boot_commands` and `test_method` job params and use a generic template, but that's not a requirement for this PR.